packages/verify-contract/README.md

```diff
@@ -116,29 +116,29 @@ verifyHardhatDeployNonTarget({
-| `kava-testnet`                                                 | `https://testnet.kavascan.com/api`               |
+| Network                                                | API URL                                        |
+|--------------------------------------------------------|------------------------------------------------|
+| avalanche, avalanche-mainnet                           | https://api.snowtrace.io/api                   |
+| fuji, avalanche-testnet                                | https://api-testnet.snowtrace.io/api           |
+| bsc                                                    | https://api.bscscan.com/api                    |
+| bsc-testnet                                            | https://api-testnet.bscscan.com/api            |
+| ethereum                                               | https://api.etherscan.io/api                   |
+| ethereum-goerli                                        | https://api-goerli.etherscan.io/api            |
+| goerli                                                 | https://api-goerli.etherscan.io/api            |
+| fantom                                                 | https://api.ftmscan.com/api                    |
+| fantom-testnet                                         | https://api-testnet.ftmscan.com/api            |
+| arbitrum                                               | https://api.arbiscan.io/api                    |
+| arbitrum-goerli                                        | https://api-goerli.arbiscan.io/api             |
+| polygon                                                | https://api.polygonscan.com/api                |
+| mumbai                                                 | https://api-testnet.polygonscan.com/api        |
+| optimism                                               | https://api-optimistic.etherscan.io/api        |
+| optimism-goerli                                        | https://api-goerli-optimistic.etherscan.io/api |
+| gnosis                                                 | https://api.gnosisscan.io/api                  |
+| zkpolygon, zkpolygon-mainnet                           | https://api-zkevm.polygonscan.com/api          |
+| base, base-mainnet                                     | https://api.basescan.org/api                   |
+| base-goerli                                            | https://api-goerli.basescan.org/api            |
+| zkconsensys, zkconsensys-mainnet, linea, linea-mainnet | https://api.lineascan.build/api                |
+| moonbeam                                               | https://api-moonbeam.moonscan.io/api           |
+| moonbeam-testnet                                       | https://api-moonbase.moonscan.io/api           |
+| kava, kava-mainnet                                     | https://kavascan.com/api                       |
+| kava-testnet                                           | https://testnet.kavascan.com/api               |
```

Comment:
Looks great! We added a corresponding `-mainnet` key for most urls here (`bsc`, `ethereum`, `fantom`, `arbitrum`, `polygon`, `optimism`, `moonbeam`). There are also more chains that are not listed in the readme at all -- do we want to include those (eg. `amoy`, `astar`, `aurora`, klaytn`, etc.

Also, no change needed -- it would be nice to have this list in alphabetical order in the readme (we recently did the same for the `DEFAULT_SCAN_API_URLS` map)

Reply:
Yes I would alphabetize these, alternatively we just link to this file so we dont need to duplicate or make multiple changes everytime we add a new chain to be supported